packages/123done/static/index.html

```diff
@@ -1,174 +1,85 @@
-            </div>
-            <div class="right-col">
-              <h3>Discover Yourself</h3>
-              <p>
-                Explore patterns in the way you work and improve your routines.
-              </p>
-            </div>
-          </div>
-          <div class="two-col clearfix">
-            <div class="right-col">
-              <h3>Do The Impossible</h3>
-              <p>Achieve your dreams.</p>
-            </div>
-            <div class="right-col">
-              <img src="/img/rocket@2x.png" alt="rocket-logo" />
-            </div>
-          </div>
-        </div>
-      </section>
-    </div>
-
-    <div id="lists">
-      <header id="header-main">
-        <div class="container">
-          <h1><span class="logo"></span><span class="title">123done <span class="pro-status">Pro!</span></span></h1>
-          <button id="logout">logout</button>
-          <div id="loggedin"><span></span></div>
-          <div id="subscriptionCTA">
-            <a href="//127.0.0.1:3030/subscriptions/products/123doneProProduct" class="btn btn-persona btn-subscribe">Subscribe for Pro</a>
-          </div>
-        </div>
-      </header>
-      <section class="todo">
-        <div class="container">
-          <div class="preroll">
-            <ul id="signinhere" style="font-size: 15pt">
-              <p>Sign in to get started making todo lists!</p>
-            </ul>
-          </div>
-
-          <h3>To Do:</h3>
-          <form
-            class="form-inline clearfix"
-            id="addform"
-            action="#"
-            method="post"
-          >
-            <div>
-              <input
-                class="input-xxlarge"
-                type="text"
-                name="newitem"
-                id="newitem"
-                placeholder="new item"
-                required
-                autocomplete="off"
-              />
-              <input class="btn" id="addtask" type="submit" value="Add" />
-            </div>
-          </form>
-          <ul id="todolist">
-            <div id="donelistwrapper">
-              <h3>Done:</h3>
-              <ul id="donelist"></ul>
-            </div>
-          </ul>
-        </div>
-      </section>
-    </div>
-
-    <footer id="footer-main">
-      <div class="container">
+<head>
+  <meta charset="UTF-8">
+  <title>Firefox Fortress</title>
+  <meta content="" type="description">
+  <meta content="IE=edge" http-equiv="X-UA-Compatible">
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <meta content="supported" name="firefox-accounts">
+  <link href="/img/favicon.png" rel="icon" type="image/png">
+  <link href="/bower_components/normalize-css/normalize.css" rel="stylesheet" type="text/css">
+  <link href="/css/main.css" rel="stylesheet" type="text/css">
+  <script src="/bower_components/jquery/dist/jquery.min.js"></script>
+</head>
+<body class="">
+  <div id="splash-page">
+    <header>
+      <div class="logo-wrapper">
```

Comment:
@johngruen Hey, it turns out that changing the top-level selectors for 123done also requires changing selectors in the selenium tests. See #1831 for a starting point, but I'm guessing still other selectors will be impacted by this change, since the signup and signin link selectors will also change here.

Reply:
Hmm, #1833 was supposed to fix things so the content server tests run when 123done changes, but I can see that didn't happen here. 😞

Reply:
Oh, I think this branch just needs rebasing against `master` actually. @johngruen, if you rebase then force push, it should fingers-crossed-touch-wood trigger the content server tests and you can see for sure whether they fail (and which ones fail).

Reply:
Just saw this comment, thanks @philbooth! I'll rebase and force-push now and see if it triggers the content tests